Meta/Lagom/Tools/CodeGenerators/LibWeb/BindingsGenerator/IDLGenerators.cpp

```diff
@@ -3418,13 +3418,63 @@ JS_DEFINE_NATIVE_FUNCTION(@class_name@::@attribute.setter_callback@)
 )~~~");
                     // FIXME: 7. If contentAttributeValue corresponds to a state of attributeDefinition with no associated keyword value, then return null.
                 }
-            } else if (attribute.type->name() != "boolean") {
+                // If a reflected IDL attribute has the type boolean:
+            } else if (attribute.type->name() == "boolean") {
```

Comment:
Unless this has changed recently, I think the standard style for this is
```suggestion
            }
            // If a reflected IDL attribute has the type boolean:
            else if (attribute.type->name() == "boolean") {
```
Same for the other changes below